common/web/utils/src/kmwstring.ts

```diff
@@ -470,6 +470,11 @@ export default function extendString() {
     p._kmwCodePointToCodeUnit = bEnable ? p.kmwCodePointToCodeUnit : p.kmwBMPCodePointToCodeUnit;
     p._kmwCodeUnitToCodePoint = bEnable ? p.kmwCodeUnitToCodePoint : p.kmwBMPCodeUnitToCodePoint;
   }
+
+  // Ensure that _all_ String extensions are established, even if disabled by default.
+  if(!String._kmwFromCharCode) {
+    String.kmwEnableSupplementaryPlane(false);
+  }
```

Comment:
Rather than auto-enable it during its definition, it double-checks that it hasn't already _been_ enabled.  If it has, there's a chance it was enabled with `true` instead of `false`, and we don't want to overwrite that.

Note that including this _specific_ module and evaluating `extendString()` is necessary for the string extensions to take effect.  As for the "done now" part of your question, that specific if-check is in case `extendString()` ends up being called multiple times.